core/src/kura.rs

```diff
@@ -306,7 +308,7 @@ impl Kura {
     #[allow(clippy::expect_used)]
     // The below lint suggests changing the code into something that does not compile due
     // to the borrow checker.
-    pub fn get_block_by_height(&self, block_height: u64) -> Option<Arc<VersionedCommittedBlock>> {
+    pub fn get_block_by_height(&self, block_height: u64) -> Option<Arc<VersionedSignedBlock>> {
```

Comment:
the thing is, only `VersionedSignedBlock` can be de/serialized because we cannot guarantee correctness for other block states unless validated against the `wsv`. What we can consider (if performance is hindered) is to introduce `CommitedBlock::assume_valid` for converting `VersionedSignedBlock` to `CommittedBlock` without any validation. 

This is only relevant when starting blockchain with `kura.get_block_by_height` in `sumeragi.mod.rs`